packages/mendel-pipeline/src/helpers/analytics/cli-printer.js

```diff
@@ -54,48 +47,59 @@ class CliPrinter extends BasePrinter {
             return Object.keys(subData).forEach(groupedName => {
                 const dataPart = subData[groupedName];
 
-                println(chalk.underline(groupedName), indentation);
+                console.log(new Array(indentation + 1).join('  ') + chalk.underline(groupedName));
                 this._print(dataPart, dimensions.slice(1), indentation + 1);
             });
         }
 
         // Group/aggregate the data
         const points = Object.keys(subData).map(groupedName => {
             const dataPart = subData[groupedName];
+
+            // Special case: make the labels pretty!
+            if (indentation === 0 && dimensions.length === 1 && dimension !== 1) {
+                // Example: <pid> (groupNameA+groupNameB+groupNameC+etc…)
+                const set = new Set();
+                dataPart.forEach(p => set.add(p.name.split(':')[1]));
+                const groupNames = Array.from(set.keys()).slice(0, 3).join('+') + (set.size > 3 ? '+etc…' : '');
+                groupedName += ` (${groupNames})`;
+            }
```

Comment:
Lol, it does not look that much difference, we were doing most calculation ourselves anyway.